packages/app/src/common/hooks/use-analytics.ts

```diff
@@ -23,9 +32,9 @@ export const useAnalytics = () => {
     });
   };
 
-  const doChangeScreen = (screen: ScreenName) => {
-    doTrackScreenChange(screen, authRequest);
-    return changeScreen(screen);
+  const doChangeScreen = (path: ScreenPaths) => {
+    doTrackScreenChange(path, authRequest);
```

Comment:
Do we still need this, now that it is actually changing url?

Reply:
We do, because it's basically a wrapper for tracking etc